lib/elixir/test/elixir/process_test.exs

```diff
@@ -170,6 +170,17 @@ defmodule ProcessTest do
     end
   end
 
+  describe "set_label/1" do
+    test "sets a process label, compatible with OTP 27+ `:proc_lib.get_label/1`" do
+      label = {:some_label, :random.uniform(99999)}
+      assert :ok = Process.set_label(label)
+
+      if String.to_integer(System.otp_release()) >= 27 do
```

Comment:
```suggestion
      if System.otp_release() >= "27" do
```

Reply:
🤔 comparing strings will be right until you get to three digits:
```
iex> "100" > "27"
false
iex> String.to_integer("100") > 27
true
```
...which is surely good enough but, seems technically less correct.

Reply:
I think the point about correctness is valid, but this should probably be addressed by exposing a new safe function and replace all other usages as well? (e.g. `@spec otp_major_version() :: integer`)

Reply:
I believe Erlang does not guarantee it will always be parsed as a string (which is potentially why they return strings?)

Reply:
Yes, although it seems to be the major version: https://erlang.org/documentation/doc-15.0-rc1/erts-14.3/doc/html/erlang.html#system_info/1-system-information in all versions we care about.

My point was to introduce such a function ourselves, which we guarantee will always return a valid integer corresponding to the major version. We could use `Integer.parse/1` if needed.

Reply:
There is no guarantee Erlang won't break it in the future and then we have a function that no longer works. :)

Reply:
That's the thing: we'll have a tested function and we should be able to figure out a way to keep it working in the future - even if Erlang changes the format under us.
If they do change, comparisons on `otp_release` will silently return nonsense, but our function will be able to address this nicely.

I'm not strongly advocating for it, just thinking aloud about how we could offer something more robust to compare the Erlang major version.